jwt.go

```diff
@@ -428,9 +428,21 @@ func stringify(val interface{}) string {
 		return stringer.String()
 	}
 
+	if slice, ok := val.([]interface{}); ok {
+		return stringify_slice(slice)
+	}
+
 	return ""
 }
 
+func stringify_slice(slice []interface{}) string {
+	var result []string
+	for _, val := range slice {
+		result = append(result, stringify(val))
+	}
+	return strings.Join(result, ", ")
```

Comment:
Since no matter what delimiter is used, actually the final functionality of stringifying an array is limited, especially when stringifying a `[]interface{}`.

However, for the very basic implementation, I'm glad to have this update merged in. But I believe a single comma is enough for this.

Reply:
- [X] use `,` as separator